Undo the TLS flags extension for negotiating RRC (#7)
tlswg/dtls-rrc: draft-ietf-tls-dtls-rrc.md

# RRC Extension

The use of RRC is negotiated via the `rrc` DTLS-only extension.  On connecting,

> the client includes the `rrc` extension in its Client Hello if it wishes to use

Reviewer: ```suggestion
the client includes the `rrc` extension in its ClientHello if it wishes to use
```

RRC.  If the server is capable of meeting this requirement, it responds with a

> `rrc` extension in its Server Hello.  The `extension_type` value for this

Reviewer: ```suggestion
`rrc` extension in its ServerHello.  The `extension_type` value for this
```

# IANA Considerations

IANA is requested to allocate an entry to the TLS `ContentType`
registry, for the `return_routability_check(TBD2)` defined in this document.
The `return_routability_check` content type is only applicable to DTLS 1.2 and
1.3.

IANA is requested to allocate the extension code point (TBD1) for the `rrc`
extension to the `TLS ExtensionType Values` registry as described in
{{tbl-ext}}.

> | Value | Extension Name | TLS 1.3 |  DTLS-Only |  Recommended | Reference |

Author: ```suggestion
| Value | Extension Name | TLS 1.3 | DTLS-Only | Recommended | Reference |
```
